site/realm/functions/getUserAdminData.js

```diff
@@ -0,0 +1,43 @@
+exports = async (source) => {
+  
+  const userApiResponse = await context.functions.execute('apiRequest', { method: 'GET', path: `/users/${source.userId}`});
+  
+    // {
+    //   "_id": "636eb28b0298163274712524",
+    //   "domain_id": "636d9086f74bbcda747d56c5",
+    //   "identities": [
+    //     {
+    //       "id": "636eb28b029816327471251a",
+    //       "provider_type": "local-userpass",
+    //       "provider_id": "636d9799d2a75aadf15afb4a",
+    //       "provider_data": {
+    //         "email": "mail@cesarvarela.com"
+    //       }
+    //     }
+    //   ],
+    //   "data": {
+    //     "email": "mail@cesarvarela.com"
+    //   },
+    //   "sessions_valid_since": {
+    //     "$numberInt": "1668199051"
+    //   },
+    //   "creation_date": {
+    //     "$numberInt": "1668199051"
+    //   },
+    //   "last_authentication_date": {
+    //     "$numberInt": "1678918007"
+    //   },
+    //   "disabled": false,
+    //   "type": "normal"
+    // }
```

Comment:
Should we remove this commented JSON object? Or at least add some explanatory comment there.